Weater-app/src/components/historyModal/HistoryModal.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { ThemeProvider } from 'styled-components';
+
+import { togglehModal, sortHandler } from '../../store/thunk/action/app';
+import { languegeObject } from '../../store/data/languageObject';
+
+import {
+  blackTheme,
+  lightTheme,
+  HistoryModalWrapper,
+} from './styledComponents/HistoryModal';
+
+const HistoryModal = () => {
+  const [orderAsc, setOrderAsc] = useState(true);
+
+  const state = useSelector((state) => state.app);
+  const { isModal, history, isBlackTheme, lang } = state;
+  const dispatch = useDispatch();
+
+  const onOrderChange = () => {
+    setOrderAsc(!orderAsc);
+    dispatch(sortHandler(orderAsc));
+  };
+
+  return (
+    <ThemeProvider theme={isBlackTheme ? blackTheme : lightTheme}>
+      <HistoryModalWrapper
+        switchModal={isModal}
```

Comment:
current file tells us that it's modal (HistoryModal.jsx ). so why we need variable `isModal` here ? maybe name of file, class is not correct? or we need to move part of code to separate file ?